Cache.js

```diff
@@ -19,8 +19,7 @@ class Cache {
   async open () {
     logger.debug('open', this.path)
 
-    if (this.store)
-      return Promise.resolve()
+    if (this.store) { return Promise.resolve() }
```

Comment:
No, it doesn't - standard just automatically does this if you use `standard --fix`. What it needs are brackets. For single line returns, I think it makes it one line, but you could make it three lines just as easily.

Reply:
Manually changed them all. Does this work better?

Reply:
Much better! 👍 The question is, then, can we make that a lint rule? :) 

I know, we can't, and that takes us a step back in automating all of linting and we'll have to pay attention to those particular lines in reviews (especially if `standard --fix` was used). Which is not ideal (and iirc was the reason to not use standard and have a way to be able to decide on these things as we go/grow). I don't think the one-liners are ideal either (in fact, quite the opposite) ...

Reply:
:/ I don't think that this is a step back, because we knew that this would be the case. This is what you said was trivial, [here](https://github.com/orbitdb/orbit-db-identity-provider/pull/9#issuecomment-444144275), when I asked if it was important to keep standard or not. 

I think that in terms of having other developers be able to use our linter without any issues, and improving the UX - one of our top priorities - this is worth it.

Reply:
It's all fine, we'll have to live with it. 

> I think that in terms of having other developers be able to use our linter without any issues, and improving the UX - one of our top priorities - this is worth it.

Having readable code > devs be able to use linter imo, but agreed that in order to finally get this done, standard gives us the best dev experience, so it's all good 👍